flow-typed/box-ui-elements.js

```diff
@@ -745,9 +745,11 @@ type TaskPayload = {
 };
 
 type TaskUpdatePayload = {
+    addedAssignees: SelectorItems,
```

Comment:
Is this right? Thought it would be `Array<TaskCollabAssignee>`

Reply:
other parts of the API code expect `SelectorItems` when adding assignees to existing tasks and creating new tasks. Leaving this be for now to de-risk changes to those existing flows